crates/ark-contracts/arkchain/src/orderbook.cairo

```diff
@@ -163,10 +168,8 @@ mod orderbook {
                 OrderType::Auction => {
                     self._create_listing_auction(order, order_type, order_hash);
                 },
-                OrderType::Offer => { self._create_listing_offer(order, order_type, order_hash); },
-                OrderType::CollectionOffer => {
-                    self._create_listing_collection_offer(order, order_type, order_hash);
-                },
+                OrderType::Offer => { self.create_offer(order, order_type, order_hash); },
+                OrderType::CollectionOffer => { self.create_offer(order, order_type, order_hash); },
```

Comment:
Collection offer and token offer share the same function?

Reply:
Yes, I've just made the token_id optional. Do you think there's something specific to the collection offer that would justify creating a separate function?